bin/test-storybook.js

```diff
@@ -103,15 +104,20 @@ async function fetchStoriesJson(url) {
 const main = async () => {
   const targetURL = sanitizeURL(process.env.TARGET_URL || `http://localhost:6006`);
   await checkStorybook(targetURL);
-  let args = process.argv.filter((arg) => arg !== '--stories-json');
 
-  if (args.length !== process.argv.length) {
+  const { jestPlaywrightOptions, storybookRunnerOptions } = getCliOptions()
```

Comment:
@yannbf You mentioned wanting feedback on variable names. These are pretty long for my liking. I'd prefer `jestOptions` and `runnerOptions` FWIW.

Reply:
Great, thanks!